docs/demos/video-links.md

```diff
@@ -0,0 +1,9 @@
+# Introduction and Demo Videos
+
+There are several on-line videos available to help you quickly understand the concept of Akri and get some quick demos.

```

Comment:
```suggestion
If you prefer to learn through videos rather than written documentation, the following is a list of informative talks and demos on Akri.
```